### PR TITLE
Admin Center changes to support Windows high contrast mode

### PR DESCRIPTION
**Pull request by lavenal**
When Windows high contrast mode is on, dojo native behavior is to enable button with text label and disable button with icon. If the icon button is able to be displayed in both white on black and black on white themes, then we will keep the icon button. Otherwise button with text label will be used. This PR addresses the changes in admin center + the explore tool.

Some screen shots with the changes:

image

image

image

image

image

image

image

### CODE REVIEW

**Comment by lavenal** [comment]
#build

**Comment by lavenal** [comment]
From the PB, the fat bucket com.ibm.ws.ui.tool.explore_fat.gui.nd.rw was run twice. All tests ran successfully one time when in full mode. When running in lite mode, testAddingRemovingColumns in ListViewTest failed. 

image

Looking at the error, it seems like an internal selenium error as it failed right away without even waiting. With that, I don't think the error is caused by changes in the PR.

**Comment by lavenal** [comment]
#build

**Comment by lavenal** [comment]
#build

**Comment by lavenal** [comment]
#run-libby-bot

**Comment by skasund** [comment]
L2 message review not required.

**Review by dmuelle — APPROVED** [review]
LGTM

File: dev/com.ibm.ws.ui.tool.explore/resources/WEB-CONTENT/jsExplore/widgets/graphs/GraphContainer.js [hunk 1]
```diff
@@ -279,7 +285,31 @@ define([
...
-        }
+        },
+
+        /**
+         * Add a class to indicate that it is high contrast black on white theme.
+         * This is to change the background color of the expand/collase graph dialog button
+         * which has a white arrow png.
+         */
+        __handleHighContrastBackgroundColor: function() {
+          console.log("----- lavena: handling black on white contrast for graphs button");
```
Comment (k8vance88) [inline dev/com.ibm.ws.ui.tool.explore/resources/WEB-CONTENT/jsExplore/widgets/graphs/GraphContainer.js]: We should remove the console.log

**Review by k8vance88 — COMMENTED** [review]
Just need to remove the console.log, otherwise looks good

File: dev/com.ibm.ws.ui.tool.explore/resources/WEB-CONTENT/jsExplore/widgets/graphs/GraphContainer.js (hunk 1 above)
Comment (lavenal) [reply dev/com.ibm.ws.ui.tool.explore/resources/WEB-CONTENT/jsExplore/widgets/graphs/GraphContainer.js]: Thank you for catching it. Just pushed a commit to remove it.

**Comment by lavenal** [comment]
#build

**Review by k8vance88 — APPROVED** [review]

**Comment by lavenal** [comment]
#run-libby-bot

**Pull request merged**